bin/main.rb

```diff
@@ -1,2 +1,52 @@
 #!/usr/bin/env ruby
 puts 'Hello World!'
```

Comment:
You can remove this from the previous milestone, not needed at this point.

Reply:
okay done